Userland/Utilities/pro.cpp

```diff
@@ -156,6 +159,8 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     DeprecatedString method = "GET";
     StringView method_override;
     HashMap<DeprecatedString, DeprecatedString, CaseInsensitiveStringTraits> request_headers;
+    DeprecatedString username;
+    DeprecatedString password;
```

Comment:
I think these can be `String`s instead of `DeprecatedString`s. (Initialized by using `String::from_utf8(credentials.substring_view(...))`)

Reply:
Thanks! Fixed just now.